tools/topology/sof-tgl-max98357a-rt5682.m4

```diff
@@ -30,6 +30,19 @@ DEBUG_START
 # PCM4 ----> volume -----> iDisp3
 # PCM5 ----> volume -----> iDisp4
```

Comment:
Can you add the graph comment for the whole topology with WoV feature here.

Reply:
Done